libs/dh/wholesale/feature-requests/src/lib/table.ts

```diff
@@ -95,7 +95,7 @@ type Request = ExtractNodeType<GetRequestsDataSource>;
 
         <ng-container *wattTableCell="columns['state']; let row">
           <dh-process-state-badge [status]="row.state">
-            {{ t('states.' + row.state) }}
+            {{ 'shared.states.' + row.state | transloco }}
```

Comment:
The translations for requests are actually slightly different, but I like just using the shared translations (hopefully UX agrees 😛). But this means that there are leftover translations in `"wholesale.requests.states"` that should be removed.

Reply:
You have more domain knowledge in wholesale, will you remove the left overs? :-)